packages/nimble-blazor/Examples/Demo.Shared/Pages/ComponentsDemo.razor

```diff
@@ -228,6 +228,22 @@
                 <NimbleListOption Value="tab-3">Tab 3</NimbleListOption>
             </NimbleSelect>
         </div>
+        <div class="sub-container">
+            <div class="container-label">Tabs - Anchor</div>
+            <NimbleAnchorTabs @bind-ActiveId="@ActiveAnchorTabId">
```

Comment:
So this sets up a two-way binding to ActiveId but that doesn't make sense right because there is not event ,etc that will update the ActiveId? Should this just be `ActiveId="@ActiveAnchorTabId"`?

Reply:
I'm actually curious, because I thought this would have created a compile-time error since there is no EventCallback defined for this property.

Reply:
Not an answer but interesting tidbit found while searching around on [avoiding side-effects in `[Parameter]` accessors](https://github.com/dotnet/aspnetcore/issues/24599#issuecomment-697588562).

Reply:
Good point. Not sure why that wasn't a compile error, but it's fixed now.